### PR TITLE
Bugs, UI Improvements, Performance Improvements

### PR DESCRIPTION
**Pull request by crishellco**
## Tickets
- https://issues.aphelion-neo.com/tktview?name=99b1494ea6
- https://issues.aphelion-neo.com/tktview?name=f09606ddb8

## Changes
- Fix formatting on various table components
- Fix date on `dashboard/RecentTransactions` component
- Call `this.$store.dispatch('fetchPortfolio')` after logging into saved wallet from settings
- Update total APH supply
- Use in-memory storage, but persist to disk using main renderer and IPC.
- Fix `tokens.tokenExists` logic
- Clean up recent transaction and history table interactions
- Fix claim button styles

### CODE REVIEW

File: src/main/services/storage.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import { ipcMain } from 'electron';
+
+const store = new Store();
+
+ipcMain.on('storage.delete', (event, key) => {
+  store.delete(key);
+});
+
+ipcMain.on('storage.set', (event, key, value) => {
+  store.set(key, value);
```
Comment (jsolman) [inline src/main/services/storage.js]: Both set and delete here in this method end up writing back to the file system synchronously for every storage event:
See
https://github.com/sindresorhus/conf/blob/master/index.js#L70
https://github.com/sindresorhus/conf/blob/master/index.js#L156

We should really be batching writes and writing back if more than an expiry time expires or if we have more than 'x' number of writes.

File: src/renderer/services/storage.js [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
+let localStore = store.store;
 
 export default {
-  clear() {
-    return store.clear();
-  },
-
   delete(key) {
-    return store.delete(key);
+    localStore = _.omit(localStore, key);
```
[the remark this answers is not in the document]
Comment (crishellco) [reply src/renderer/services/storage.js]: There are probably better ways, but we honestly don't even use this method currently, so I'm not too worried about the performance of it.

File: src/main/services/storage.js (hunk 1 above)
Comment (crishellco) [reply src/main/services/storage.js]: Writes happen in the main process via the IPC, so it won't affect the performance of the UI as they are on different processes/threads.

Comment (jsolman) [reply src/main/services/storage.js]: Yes, this comment was not about affecting performance in the UI thread; my concern here is excessive writes to disk. For example, ten events in rapid succession writes the whole file 10 times. Do we know how often we are triggering these events and the impact of this once the storage size gets bigger. We are saving transaction history in storage, correct?

**Comment by jsolman** [comment]
These are great improvements. If we decide to batch writes we can do that on top of these changes.

**Pull request merged**